fedimint-api/src/bitcoin_rpc.rs

```diff
@@ -67,3 +80,11 @@ pub fn fm_electrum_rpc_env_value_to_url(value: &OsStr) -> anyhow::Result<Url> {
         anyhow::format_err!("Url not ascii text")
     })?)?)
 }
+
+// TODO: (@oleonardolima) Should we use a single fn instead of 3, such as: `fm_backend_rpc_env_value_to_url` ?
```

Comment:
Yeah. One function for all, then `FM_BITCOIND_RPC_DEFAULT_FALLBACK` moved to the `else` in `select_bitcoin_backend_from_envs`